tests/FSharp.Compiler.Private.Scripting.UnitTests/FSharpScriptTests.fs

```diff
@@ -65,26 +65,21 @@ type InteractiveTests() =
     [<Test>]
     member __.``Assembly reference event successful``() =
         use script = new FSharpScript()
-        let testAssembly = "System.dll"
-        let mutable assemblyResolveEventCount = 0
-        let mutable foundAssemblyReference = false
-        Event.add (fun (assembly: string) ->
-            assemblyResolveEventCount <- assemblyResolveEventCount + 1
-            foundAssemblyReference <- String.Compare(testAssembly, Path.GetFileName(assembly), StringComparison.OrdinalIgnoreCase) = 0)
-            script.AssemblyReferenceAdded
-        script.Eval(sprintf "#r \"%s\"" testAssembly) |> ignoreValue
-        Assert.AreEqual(1, assemblyResolveEventCount)
-        Assert.True(foundAssemblyReference)
+        let testCode = """
+#r "System.dll"
+let stacktype= typeof<System.Collections.Stack>
```

Comment:
I'd argue this test can be deleted.  If I recall, it was added purely to test the event which is now gone.